pkg/sloop/webfiles/sloop_ui.js

```diff
@@ -5,16 +5,34 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-const displayMaxX = document.documentElement.clientWidth;
-const displayMaxY = document.documentElement.clientHeight;
+const palette = {
+        baseDark: ["#2E3440", "#3B4252", "#434C5E", "#4C566A"],
+        baseLight: ["#D8DEE9", "#E5E9F0", "#ECEFF4"],
+        primary: ["#8FBCBB", "#88C0D0", "#81A1C1", "#5E81AC"],
+        highlight: ["#BF616A", "#D08770", "#EBCB8B", "#A3BE8C",
+            "#B48EAD","#ADA8B6","#da7650","#D496A7","#ABC4AB",
+            "#A4A6D2","#91BEF2","#97B1A6","#8A9B68"]
+};
+
+
+// Globals Live Here
+let topAxis, bottomAxis, xAxisScale, yAxisBand, data, theTime, axisTop, axisBottom, smallBarOffset, displayMaxX, displayMaxY;
```

Comment:
would it help future developers if these were on separate lines with a comment explaining their use (is the name sufficient)?

Reply:
Yesssssss I think it will. I'll go through and explain these.

Reply:
This is a lot though- I'll probably do that on a separate PR.

Reply:
this will never be easier then now, but shouldn't be a PR blocker